### PR TITLE
Upstream multicore hello world

### PR DESCRIPTION
**Pull request by gmarull**

### CODE REVIEW

**Review by grochu — APPROVED** [review]

**Comment by gmarull** [comment]
> Hi @gmarull , can you tell who has agreed to that?

I guess the question is: why maintain a sample in NCS that has an upstream equivalent?

**Comment by pdunaj** [comment]
> > Hi @gmarull , can you tell who has agreed to that?
> 
> I guess the question is: why maintain a sample in NCS that has an upstream equivalent?

Convenience - it's easier for us. I expect I don't have a problem with migrating that if you take the ownership for it. We have other stuff to do and I see @grochu is already ok.

@carlescufi, will you take ownership for this sample?

**Review by pdunaj — DISMISSED** [review]
Let's confirm the ownership first.

**Comment by gmarull** [comment]
> > > Hi @gmarull , can you tell who has agreed to that?
> > 
> > 
> > I guess the question is: why maintain a sample in NCS that has an upstream equivalent?
> 
> Convenience - it's easier for us. I expect I don't have a problem with migrating that if you take the ownership for it. We have other stuff to do and I see @grochu is already ok.
> 
> @carlescufi, will you take ownership for this sample?

I guess maintenance can be taken, it's actually easier to maintain upstream as it will get updated as needed so it'll never be broken.

**Review by greg-fer — CHANGES_REQUESTED** [review]
Requires a changelog entry about the sample removal.

**Comment by gmarull** [comment]
> Requires a changelog entry about the sample removal.

done

File: doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst [hunk 1]
```diff
@@ -570,6 +570,8 @@ Multicore samples
 
 |no_changes_yet_note|
 
+* Multicore hello world sample removed in favor of :zephyr:code-sample:`sysbuild_hello_world`, which has equivalent functionality.
```
Comment (greg-fer) [inline doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst:573]:
```suggestion
* Removed the "Multicore Hello World application" sample in favor of :zephyr:code-sample:`sysbuild_hello_world`, which has equivalent functionality.
  This also removes the Multicore samples category from the :ref:`samples` page.
```

**Review by greg-fer — CHANGES_REQUESTED** [review]
Just an edit to the changelog.

**Comment by gmarull** [comment]
@greg-fer @pdunaj ping

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 2]
```diff
@@ -381,20 +381,20 @@ To transition the LCS to ``RoT``, do the following:
...
 
 To build and program the sample to the nRF54H20 DK, complete the following steps:
 
 1. Connect the nRF54H20 DK to your computer using the **DEBUGGER** port on the DK.
-#. Navigate to the :file:`nrf/samples/multicore/hello_world` folder containing the sample.
+#. Navigate to the :file:`zephyr/samples/sysbuild/hello_world` folder containing the sample.
 #. Build the sample for application and radio cores by running the following command::
 
-      west build -p -b nrf54h20dk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20dk_cpuapp_cpurad .
+      west build -p -b nrf54h20dk_nrf54h20_cpuapp -T sample.sysbuild.hello_world.nrf54h20dk_cpuapp_cpurad .
```
Comment (FrancescoSer) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:393]: Could you update the board target in the command here?

File: doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst [hunk 3]
```diff
@@ -381,20 +381,20 @@ To transition the LCS to ``RoT``, do the following:
...
 
 #. Alternatively, build the sample for the application and PPR cores by running the following command::
 
-      west build -p -b nrf54h20dk_nrf54h20_cpuapp -T sample.multicore.hello_world.nrf54h20dk_cpuapp_cpuppr .
+      west build -p -b nrf54h20dk_nrf54h20_cpuapp -T sample.sysbuild.hello_world.nrf54h20dk_cpuapp_cpuppr .
```
Comment (FrancescoSer) [inline doc/nrf/device_guides/working_with_nrf/nrf54h/ug_nrf54h20_gs.rst:397]: same here

**Review by FrancescoSer — CHANGES_REQUESTED** [review]

**Comment by gmarull** [comment]
@greg-fer @FrancescoSer ping

**Review by FrancescoSer — APPROVED** [review]
Approving but please implement the small suggestion

**Pull request merged**